android-kit-base/src/test/kotlin/com/mparticle/kits/KitConfigurationTest.kt

```diff
@@ -450,7 +450,7 @@ class KitConfigurationTest {
         consentForwardingRule.put("v", ruleArray)
         val rule1 = JSONObject()
         ruleArray.put(rule1)
-        rule1.put("h", KitUtils.hashForFiltering("1" + "foo purpose"))
+        rule1.put("h", HashingUtility.hashGDPRContentPurposeKey("1" + "foo purpose"))
```

Comment:
```suggestion
        rule1.put("h", HashingUtility.hashGDPRContentPurposeKey("foo purpose"))
```
Shouldn't the `"1" +` be removed here since the new method adds it?